### ISSUE TITLE
Guest entries 4 and Craft 5 support

### ISSUE THREAD

**Issue by jannisborgers** [bhashkar007/guest-entries-notification#14]
A wonderful plugin that adds essential functionality to the Guest Entries plugin. Thank you!

Do you plan on updating the plugin to support Guest Entries 4 and Craft 5? Guest Entries 4 is the latest version for Craft 4, but I cannot update from Guest Entries 3 because your plugin requires version 3 of that plugin.

As a workaround I copied the repo to `[project root]/plugins-local` and made the following changes to the local plugin’s `composer.json` file:

```json
{
    "version": "3.0.0",
    "require": {
        "craftcms/guest-entries": "^4.0.0"
    },
}
```

Then I updated the project’s `composer.json` like so:

```json
{
    "require": {
        "craftcms/guest-entries": "^4.0.0",
        "by/guest-entries-notification": "^3.0"
    },
    "repositories": [
        {
          "type": "path",
          "url": "./plugins-local/guest-entries-notification"
        }
    ]
}
```

Testing the basic functionality (without template overrides), the plugin works just the same as before 🎉

I suspect but cannot verify that changing the `craftcms/cms` dependency to `^5.0.0` in the project’s `composer.json` would make it work with Craft 5 as well. I have other plugins in this project that cannot be updated to Craft 5 yet, so I’ll have to wait until the developers update them.

**Comment by CopulaTech** [comment]
I'd like to add my support for an update, please.

**Comment by amphibian** [comment]
Yes, please make this simple change to allow us to properly update the plugin!

**Comment by bhashkar007** [comment]
@amphibian I 'll try to make all these update in next few days.

**Comment by engram-design** [comment]
@bhashkar007 I'm not sure if you're still developing this plugin, but as part of our [Craft 4-5 migration project](https://verbb.io/craft-4), we've had a number of people request this plugin be available for Craft 5. As such, we'd be keen to continue development at Verbb if you're no longer working on this plugin.

Of course, if you've got plans to add this support, feel free to ignore this request!

**Comment by amphibian** [comment]
Bump on this - any chance of getting this resolved? Bit me again doing a Craft update today.

**Comment by amici-infotech** [comment]
We are doing craft 5 upgrade too of numerous sites and wanted to make this work :(
